runtime/syntax/pamconf.vim

```diff
@@ -11,7 +11,7 @@ endif
 let s:cpo_save = &cpo
 set cpo&vim
 
-syn match   pamconfService          '^[[:graph:]]\+'
+syn match   pamconfService          '^-?[[:graph:]]\+'
```

Comment:
Shouldn't it be `-\?`?
```suggestion
syn match   pamconfService          '^-\?[[:graph:]]\+'
```

BTW, have you contacted the maintainer?

Reply:
Oh, yes. Thanks for the heads up.

I haven't contacted the maintainer. I'll try to do it now.

Reply:
Well, I actually replied without trying. I checked the document, it says `\?` under `pattern-multi-items`. But it only works when there isn't a backslash before `?`.

Reply:
Now I'm confused.
`[[:graph:]]` matches `-`. So, `pamconfService` should match `foobar` and also `-foobar` even without your fix.
What do you really want to fix?

Reply:
You can try the following snippet:
```
#%PAM-1.0

auth            include         system-login
-auth           optional        pam_gnome_keyring.so
password        include         system-login
-password       optional        pam_gnome_keyring.so use_authtok
session         include         system-login
-session        optional        pam_kwallet5.so auto_start
```

What I want to achieve is that highlighting are more or less the same for lines starting with or without `-`.

Reply:
What you did is making `pamconfService` not to match the keywords.
It is almost the same as commenting out the `pamconfService` lines.

```vim
" syn match   pamconfService          '^[[:graph:]]\+'
"                                     \ nextgroup=pamconfType,
"                                     \ pamconfServiceLineCont skipwhite
```